Config/dev_env/types.ts

```diff
@@ -0,0 +1,36 @@
+export type entryObj =
+  {
+    id: number,
+    mainLine: string,
+    type: gutkaEntry,
+    parentGutka: string,
+  };
+export type storedGutka = {
+  items: entryObj[],
+  name: string,
+}
+export type gutkaEntry = 'Shabad' | 'Bani';
+export type SearchType = 0 | 1 | 2 | 3 | 4;
+export type QueryType = 'Shabad' | 'Bani';
+
+export type GutkaApi = {
+  gutkaNames: string[],
```

Comment:
actually scratch the test comment what we can do is in the context reducer do this:
```javascript
<Provider value={{value}}> 
```